### DISCUSSION TITLE
Environment variable and secret storage

### DISCUSSION THREAD

**Discussion by noahstreller** [coollabsio/coolify#2620] [Feature Requests]
Hi! I recently migrated from Vercel to Coolify and I am very happy about the outcome. However, using the Vercel CLI I was able to run `vercel env pull .env.development.local` to obtain my env file, and Coolify doesn't seem to have an equivalent.

It would be amazing if Coolify could feature such env/secret management as well. 

Or is there maybe a different alternative to the `vercel env` command?

**Comment by noahstreller** [comment]
Ended up self hosting `Infisical` due to lack of activity here. Works alright for me.

**Comment by JPustkuchen** [comment]
Maybe Infisical is interesting for integration into coolify one day?
